### ISSUE TITLE
A11y_VisualStudioCodeClient_Customizations and Adding Extensions_keyboard :Focus order is incorrect as it first lading on the categories at right and then moves to the details at left.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141712]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #WCAG2.1.1; #Keyboard; #FocusOrder; #AINotInScope; #FTP; #STP; 

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1.Open Visual studio code editor.
2.Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.Using Tab key navigate to the "More" option and select different options( "Show installed Extension, Show outdated extension, Show popular Extension,Etc")
4.Click on any installed extension
5.Try to read all the details about specific extensions.
6.Verify whether the keyboard focus first lands on catagories and then shift to details (i.e right to left) .
### Actual Result:
Focus order is incorrect as it first lading on the categories at right and then moves to the details at left.

### Expected Result:
Focus order should be correct as it first lading on the details at left and then moves to the categories at right.

### User Impact
Keyboard User will not understand the focus order of the window.

**Comment by isidorn** [comment]
Can not reproduce. The focus order for me is first Details than Categories.
Please try it with latest VS Code insiders
